packages/firebase_auth/lib/firebase_auth.dart

```diff
@@ -192,4 +192,28 @@ class FirebaseAuth {
     final FirebaseUser currentUser = new FirebaseUser._(data);
     return currentUser;
   }
+
+  /// Links google account with current user and returns [Future<FirebaseUser>]
+  ///
+  /// throws [PlatformException] when
+  /// 1. No current user provided (user has not logged in)
+  /// 2. No google credentials were found for given [idToken] and [accessToken]
+  /// 3. Google account already linked with another [FirebaseUser]
+  /// See FIRAuthErrors for a list of error codes that are common to all API methods.
```

Comment:
Maybe add a link to the list of error codes? Also, document how this method exposes the error codes?

Reply:
This needs clarification: Where do you find `FIRAuthErrors`? Where do I get the error code from?

Reply:
I added references to android and ios docs. In FirebaseAuth PlatformExceptions don't provide specific error codes from what I know so I mentioned it in TODO.